src/Our.Umbraco.NestedContent/Web/UI/App_Plugins/NestedContent/Js/nestedcontent.controllers.js

```diff
@@ -423,6 +426,18 @@ angular.module("umbraco").controller("Our.Umbraco.NestedContent.Controllers.Nest
             }
             return _p8() + _p8(true) + _p8(true) + _p8();
         };
+
+        var getCurrentNodeIndex = function (node) {
+            return node.id === $scope.realCurrentNode.id;
+        }
+
+        var backupGridConfig = function (prop) {
+            return prop.editor == "Umbraco.Grid" ? prop : false;
```

Comment:
totally agree, we should not be tying nested content to the grid property editor so some other solution is needed. I'm not sure what we can do with events from the angular side though.